### DISCUSSION TITLE
[FR] Add 'Copy reference tag' action to entries element index

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/cms#12765] [Ideas]
A client discovered a 'Copy reference tag'  element action on the asset index pages, but missed it on the entries index page.

Could add it via the `EVENT_REGISTER_ACTIONS` event, would be nice to have it out of the box though.

Thanks /Klara

**Comment by MangoMarcus** [comment]
Would be good to have this

**Reply by wsydney76** [reply]
I think this has simply been overlooked, for now you can register this action in a module yourself:

```php
use craft\base\Element;
use craft\elements\actions\CopyReferenceTag;
use craft\elements\Entry;
use craft\events\RegisterElementActionsEvent;
use yii\base\Event;

....

Event::on(
    Entry::class,
    Element::EVENT_REGISTER_ACTIONS,
    function(RegisterElementActionsEvent $event) {
        $event->actions[] = CopyReferenceTag::class;
    }
);
```

**Reply by MangoMarcus** [reply]
@wsydney76 Nice! Ty
